contracts/src/core/EigenDACertVerifier.sol

```diff
@@ -18,12 +18,24 @@ import "../interfaces/IEigenDAStructs.sol";
  */
 contract EigenDACertVerifier is IEigenDACertVerifier {
 
+    /// @notice The EigenDAThresholdRegistry contract address
     IEigenDAThresholdRegistry public immutable eigenDAThresholdRegistry;
+
+    /// @notice The EigenDABatchMetadataStorage contract address
+    /// @dev On L1 this contract is the EigenDA Service Manager contract
     IEigenDABatchMetadataStorage public immutable eigenDABatchMetadataStorage;
+
+    /// @notice The EigenDASignatureVerifier contract address
+    /// @dev On L1 this contract is the EigenDA Service Manager contract
```

Comment:
Do these notices show up somewhere in some auto-generated documentation or something? I'm generally very much against these kind of useless comments that provide absolutely no information, other than restating the var name + type. 

@notice in my mind should provide external context. We had talked about adding information about which contracts are for V1 vs V2. You did not add this information. Why?